xradar/io/backends/iris.py

```diff
@@ -218,7 +218,10 @@ def decode_kdp(data, **kwargs):
     See 4.4.20 p.77
     """
     wavelength = kwargs.pop("wavelength")
-    zero = data[data == -128]
+    zero = data == -128
```

Comment:
@egouden You mentioned the difference between "no data available" and "area not scanned". We might not have good solution to this, but would it make sense here to bind those two together and set to nan?